terraform-plan/README.md

```diff
@@ -1,42 +1,69 @@
-| `output`                      | false    | compact | Set the output format for scan results: compact, yaml, json, junit, csv, summary, full, report (default "compact")                                                                                                               |
-| `path`                        | true     |         | Path to the Terraform working directory.                                                                                                                                                                                         |
-| `score-threshold`             | false    | 0       | Sets the score threshold for scans. Scores that fall below the threshold will exit 1. (default "0" - job continues regardless of the score returned by a scan).                                                                  |
-| `service-account-credentials` | false    |         | Base64 encoded [service account credentials](https://mondoo.com/docs/platform/service_accounts/#creating-service-accounts) used to authenticate with Mondoo Platform. You can also use the environment variable mentioned below. |
+| Property                      | Required | Default     | Description                                                                                                                                                                                                                      |
+| ----------------------------- | -------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `log-level`                   | false    | info        | Sets the log level: error, warn, info, debug, trace (default "info")                                                                                                                                                             |
+| `output`                      | false    | compact     | Set the output format for scan results: compact, yaml, json, junit, csv, summary, full, report (default "compact")                                                                                                               |
+| `path`                        | false    | ./terraform | Path to the Terraform working directory (default "./terraform")                                                                                                                                                                  |
+| `path-file`                   | false    | plan.json   | Name of plan file to scan (default "plan.json")                                                                                                                                                                                  |
```

Comment:
Why do we need both `path` and `path-file`. If we want to use both we need to rename path to directory or something.